scraper/igempnp/spiders/MemberSpider.py

```diff
@@ -0,0 +1,46 @@
+import scrapy
+from scrapy.contrib.spiders import CrawlSpider, Rule
+from scrapy.contrib.linkextractors import LinkExtractor
+
+from igempnp.items import MemberItem, TrackItem
+
+class MemberSpider(CrawlSpider):
+
+    dont_filter = True
+
+    name = "track"
+    allowed_domains = ["igem.org"]
+    start_urls = [
+        "http://igem.org/Team_List?year=2007"
+    ]
+
```

Comment:
if this is gonna be length 1 for now, lets keep it as this: 
`start_urls = ["http://igem.org/Team_List?year=2007"]`
if this is gonna be a long list, I think the convention is to do it like this:
```
start_urls = ["http://igem.org/Team_List?year=2007",
              "item2",
              "etc]
```